lib/vdsm/storage/lvm.py

```diff
@@ -577,74 +577,117 @@ def _getVGDevs(self, vgNames):
                 devices = tuple(devices)
         return devices
 
-    def _reloadvgs(self, vgName=None):
+    def _run_vgs_cmd(self, vgNames):
         cmd = list(VGS_CMD)
+        cmd.extend(vgNames)
+
+        return self.run_command_error(
+            cmd, devices=self._getVGDevs(vgNames))
+
+    def _updatevgs(self, lvm_output, vgs):
+        """
+        Update cached VGs based on the output of the LVM command.
+        Remove stale VGs (have not been updated) out of the 'vgs' list, or
+        the complete list of cached VGs if the command was run for all VGs.
+        Return list of all updated VGs.
+        """
+        assert self._lock.locked
+        upd_vgs = {}
```

Comment:
...to make it fit in one line when used in a generator below. Anyway, I will revert it to `updatedVGs`, was its original name, so that I restrain additional changes.